From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jake Bailey <5341706+jakebailey@users.noreply.github.com>
Date: Wed, 19 Oct 2022 13:59:15 -0700
Subject: [PATCH] Restore template literals, remove let/const comment

The issue with ts.preProcessFile has been fixed; monaco should get the
fix at the same time they bump to 5.0.

Remove the let const comment; I think it's better to not downlevel at
all.
---
 Herebyfile.mjs | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Herebyfile.mjs b/Herebyfile.mjs
index 7913984824..d380908e33 100644
--- a/Herebyfile.mjs
+++ b/Herebyfile.mjs
@@ -196,11 +196,7 @@ async function runEsbuild(entrypoint, outfile, taskOptions = {}) {
             "source-map-support",
         ],
         supported: {
-            // "const-and-let": false, // https://github.com/evanw/esbuild/issues/297
             "object-rest-spread": false, // Performance enhancement, see: https://github.com/evanw/esbuild/releases/tag/v0.14.46
-            // TODO(jakebailey): workaround for https://github.com/microsoft/TypeScript/issues/51072
-            // Remove once 4.9 is out with the fix and monaco is updated.
-            "template-literal": false,
         },
         logLevel: "warning",
         // legalComments: "none", // If we add copyright headers to the source files, uncomment.
-- 
2.38.0

